src/server/scripts/Spells/spell_item.cpp

```diff
@@ -4075,6 +4075,45 @@ class spell_item_greatmothers_soulcatcher : public SpellScriptLoader
     }
 };
 
+enum Eggnog
+{
+    SPELL_EGG_NOG_REINDEER = 21936,
+    SPELL_EGG_NOG_SNOWMAN = 21980,
```

Comment:
```suggestion
    SPELL_EGG_NOG_REINDEER = 21936,
    SPELL_EGG_NOG_SNOWMAN  = 21980,
```

Reply:
Very cosmetic, but shouldn't lists in C++ have their "=" lined up?
Not really sure

Reply:
Ty! Yes it's true updated Ty!